src/dnx.windows/dnx.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+
+int _tmain(int argc, _TCHAR* argv[])
+{
+    OSVERSIONINFO version_info;
+    ZeroMemory(&version_info, sizeof(OSVERSIONINFO));
+    version_info.dwOSVersionInfoSize = sizeof(OSVERSIONINFO);
+
+#pragma warning(disable:4996)
+    GetVersionEx(&version_info);
```

Comment:
Is there a reason we don't use the Version Helper functions, since this is deprecated?